theseus_gui/src/components/ui/AccountsCard.vue

```diff
@@ -4,7 +4,7 @@
     ref="button"
     class="button-base avatar-button"
     :class="{ expanded: mode === 'expanded' }"
-    @click="showCard = !showCard"
+    @click="toggleShow()"
```

Comment:
I don't think the issue mentioned is worth fixing, deleting accounts is a low cost action (someone can just log back in). I think we should revert the changes in this file